libs/go-kibana-rest/kbapi/api_test.go

```diff
@@ -73,7 +75,7 @@ func (s *KBAPITestSuite) SetupSuite() {
 
 func (s *KBAPITestSuite) SetupTest() {
 
-	// Do somethink before each test
+	// Do something before each test
```

Comment:
👏 best change in this PR.

Reply:
I'm sorry, couldn't resist to fix typo. Please, let me know if I should not do it 🙈

Reply:
Okey, googled it and reverted.

Reply:
> I'm sorry, couldn't resist to fix typo. Please, let me know if I should not do it 🙈

You did absolutely nothing wrong, sorry for the confusion. I'm 100% on board with fixing this typo, I assume we inherited it when this code was vendored in here.